052-k8s-server-side-apply.md

```diff
@@ -0,0 +1,35 @@
+<!-- This template is provided as an example with sections you may wish to comment on with respect to your proposal. Add or remove sections as required to best articulate the proposal. -->
+
+# Kubernetes Server Side Apply
+
+Make the Topic Operator use [Kubernetes Server Side Apply](https://kubernetes.io/docs/reference/using-api/server-side-apply). This will enable third parties (e.g. Kyverno, Kopf) to have their own labels/annotations on Strimzi owned Kubernetes Resources without Strimzi attempting to remove them.
```

Comment:
The name suggests a generic proposal which is about everything ... but here you talk about Topic Operator. That seems to be conflicting information. I wonder what is really the intention.

Reply:
Good point, modified to `Operators` as the change is in the `operator-common` folder as also added to the proposal